test/functional/change_stream.test.js

```diff
@@ -99,6 +100,18 @@ function triggerResumableError(changeStream, delay, onClose) {
  * @param {Function} callback
  */
 function waitForStarted(changeStream, callback) {
```

Comment:
now supports a promise version of the same thing, races a promise that rejects after 2seconds and the init event.